stdgo/reflect/Reflect.hx

```diff
@@ -975,10 +1218,26 @@ class _HaxeValue__extension {
 	}
 }
 
+// StdGo2hx holds some of the standard fields in a go2hx generated class
+class StdGo2hx {
```

Comment:
Should probably be renamed something more standard, Such as BaseType and be an interface rather then a Class.

Reply:
Since `reflect` needs to contain classes that mimic the go2hx generated classes, a `BaseType` interface would be helpful to ensure the right standard is being met.

Reply:
The current implementation of BaseType:
```
extern interface BaseType {
	public static final _typeName_:String;
}
```
...is not quite right, because of the `final`, implementing BaseType with a super/extends class means that the value would need to be passed in on `new()` - so it could not be final.